spec/jobs/ingest_archival_media_bag_job_spec.rb

```diff
@@ -43,12 +43,12 @@
       expect(file_set.transfer_notes.first).to start_with "Side A"
     end
 
-    it "creates one MediaResource for the barcode and one for the component ID" do
-      expect(query_service.find_all_of_model(model: MediaResource).size).to eq 2
+    it "creates one ScannedResource for the barcode and one for the component ID" do
```

Comment:
In some of these comments it would maybe be nice to say "Recording" instead of "ScannedResource". Which leads me to wonder, should we be testing that these have the "recording" change set?

Reply:
Added a couple checks for that, and changed instances of ScannedResource in descriptions to Recording.